lb3/task3/Makefile

```
@@ -0,0 +1,86 @@
+CС=gcc
+CFLAGS= -Wall -Werror -Wextra
+
+SOURCES=lb3task3.c # Тут вы перечислятете файлы с вашими лабами по 1 файлу на задание
+TEST_TEMPLATES_DIR=/mnt/c/Users/Вероника/Documents/GitHub/FundAlg3Sem/test_templates
```

Опять абсолютный путь(
```suggestion
TEST_TEMPLATES_DIR=../../test_templates
```